docs/standard/parallel-programming/chaining-tasks-by-using-continuation-tasks.md

```diff
@@ -1,150 +1,175 @@
-  
- Continuations are relatively easy to use, but are nevertheless powerful and flexible. For example, you can:  
-  
-- Pass data from the antecedent to the continuation.  
-  
-- Specify the precise conditions under which the continuation will be invoked or not invoked.  
-  
-- Cancel a continuation either before it starts or cooperatively as it is running.  
-  
-- Provide hints about how the continuation should be scheduled.  
-  
-- Invoke multiple continuations from the same antecedent.  
-  
-- Invoke one continuation when all or any one of multiple antecedents complete.  
-  
-- Chain continuations one after another to any arbitrary length.  
-  
-- Use a continuation to handle exceptions thrown by the antecedent.  
-  
-## About continuations  
- A continuation is a task that is created in the <xref:System.Threading.Tasks.TaskStatus.WaitingForActivation> state. It is activated automatically when its antecedent task or tasks complete. Calling <xref:System.Threading.Tasks.Task.Start%2A?displayProperty=nameWithType> on a continuation in user code throws an <xref:System.InvalidOperationException?displayProperty=nameWithType> exception.  
-  
- A continuation is itself a <xref:System.Threading.Tasks.Task> and does not block the thread on which it is started. Call the <xref:System.Threading.Tasks.Task.Wait%2A?displayProperty=nameWithType> method to block until the continuation task finishes.  
-  
-## Creating a continuation for a single antecedent  
- You create a continuation that executes when its antecedent has completed by calling the <xref:System.Threading.Tasks.Task.ContinueWith%2A?displayProperty=nameWithType> method. The following example shows the basic pattern (for clarity, exception handling is omitted). It executes an antecedent task, `taskA`, that returns a <xref:System.DayOfWeek> object that indicates the name of the current day of the week. When the antecedent completes, the continuation task, `continuation`, is passed the antecedent and displays a string that includes its result.
+
+# Chaining tasks using continuation tasks
+
+In asynchronous programming, it is common for one asynchronous operation, on completion, to invoke a second operation, and pass data to it. Traditionally, continuations have been done by using callback methods. In the Task Parallel Library, the same functionality is provided by _continuation tasks_. A continuation task (also known just as a continuation) is an asynchronous task that is invoked by another task, which is known as the _antecedent_, when the antecedent finishes.
```

Comment:
Instead of "pass data to it", how about something regarding the second operation *consuming* the results of the first asynchronous operation. If you just want to "pass data", you could start both async operations immediately.